### DISCUSSION TITLE
Every page outside of root showing 404

### DISCUSSION THREAD

**Discussion by madhawk1296** [vercel/next.js#49618] [App Router]
I'm just starting with NextJS 13. Every time I create a folder structure like `app/collection/page.tsx` is shows a 404. Has anyone else had this problem when starting?

**Comment by icyJoseph** [comment]
Nope, I have not experienced this.

Could you share your folder structure? and Next.js version?

**Comment by volpster2100** [comment]
Do you have app/layout.tsx? That’s required for the app router to work

**Comment by mandorakannu** [comment]
I think you should open the link in browser like this: `http://localhost:3000/Collection/ `
In this link the `/Collection` word The first letter is capital but you are creating the folder name in lowercase
I hope its help
